content/post/kubedb-v2024.01.31/index.md

```diff
@@ -0,0 +1,470 @@
+
+Here's a sample yaml for provisioning Solr using KubeDB.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Solr
+metadata:
+  name: solr-cluster
+  namespace: demo
+spec:
+  version: 9.4.1
+  terminationPolicy: Delete
+  zookeeperRef:
+    name: zk-cluster
+  topology:
+    Overseer:
+      replicas: 1
+      storage:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 3Gi
+        storageClassName: standard
+    data:
+      replicas: 2
+      storage:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 3Gi
+        storageClassName: standard
+    Coordinator:
+      replicas: 1
+      storage:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 3Gi
+        storageClassName: standard
+```
+
+#### supported versions
+`9.4.1` and `8.11.2`
+
+# Singlestore:
+This release introduces support for **SingleStore**, a distributed SQL database for real-time analytics, transactional workloads, and operational applications. With its in-memory processing and scalable architecture, SingleStore enables organizations to achieve high-performance and low-latency data processing across diverse data sets, making it ideal for modern data-intensive applications and analytical workflows. This release is bring support for Provisioning Singlestore in both standalone and cluster mode, Custom Configuration using kubernetes secret and Initialization using script.
+
+You can provision Singlestore cluster in production ideal cluster mode using the following YAML.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Singlestore
+metadata:
+  name: singlestore-sample
+  namespace: demo
+spec:
+  version: "8.1.32"
+  topology:
+    aggregator:
+      replicas: 3
+      storage:
+        storageClassName: "standard"
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+    leaf:
+      replicas: 2
+      storage:
+        storageClassName: "standard"
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 10Gi
+  licenseSecret:
+    name: license-secret
+  storageType: Durable
+  terminationPolicy: WipeOut
+```
+
+#### supported versions
+`3.1.32`
+
+# Pgpool
+
+This release also introduces support for **Pgpool**, an advanced connection pooling and load balancing solution for PostgreSQL databases. It serves as an intermediary between client applications and PostgreSQL database servers, providing features such as connection pooling, load balancing, query caching, and high availability. This release includes support for Provisioning, Custom Configuration using kubernetes secret and Postgres users synchronization.
```

Comment:
Custom configuration is given in initConfig field in Pgpool and operator will create a secret based on given and default configuration.